src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -35,51 +55,166 @@ void getLines() {
+    }
+
+    private ExtractableResponse<Response> 지하철들_노선_목록_조회_요청() {
+        return RestAssured.given().log().all()
+                .when()
+                .get("/lines")
+                .then().log().all()
+                .extract();
+    }
+
+    private ExtractableResponse<Response> 지하철_노선_목록_조회_요청(final ExtractableResponse<Response> createResponse) {
+        final Long id = 지하철_노선_ID_추출(createResponse);
+        return RestAssured.given().log().all()
+                .when()
+                .get("/lines/" + id)
+                .then().log().all()
+                .extract();
+    }
+
+    private ExtractableResponse<Response> 지하철_노선_수정_요청(final ExtractableResponse<Response> createResponse, final String color
+            , final String name) {
+        final Long id = 지하철_노선_ID_추출(createResponse);
+        Map<String, String> params = new HashMap<>();
+        params.put("color", color);
+        params.put("name", name);
+
+        return RestAssured.given().log().all()
+                .body(params)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when()
+                .put("/lines/"+id)
+                .then().log().all()
+                .extract();
+    }
+
+    private ExtractableResponse<Response> 지하철_노선_제거_요청(final ExtractableResponse<Response> createResponse) {
+        final Long id = 지하철_노선_ID_추출(createResponse);
+        return RestAssured.given().log().all()
+                .when()
+                .delete("/lines/"+id)
+                .then().log().all()
+                .extract();
+    }
+
+    private Long 지하철_노선_ID_추출(final ExtractableResponse<Response> createResponse1) {
+        return Arrays.asList(createResponse1).stream()
+                .map(it -> Long.parseLong(it.header("Location").split("/")[2]))
+                .findFirst().get();
+    }
+
+    private void 지하철_노선_생성됨(ExtractableResponse<Response> response) {
+        assertAll(
+                () -> assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value()),
+                () -> assertThat(response.header("Location")).isNotBlank()
+        );
+
+    }
+
+    private void 지하철_노선_생성_실패됨(ExtractableResponse<Response> response) {
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
+    }
+
+    private void 지하철_노선_목록_응답됨(ExtractableResponse<Response> response) {
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+    }
+
+    private void 지하철_노선_목록_포함됨(ExtractableResponse<Response> createResponse1, ExtractableResponse<Response> createResponse2, ExtractableResponse<Response> response) {
+        List<Long> expectedLineIds = Arrays.asList(createResponse1, createResponse2).stream()
+                .map(it -> Long.parseLong(it.header("Location").split("/")[2]))
+                .collect(Collectors.toList());
+        List<Long> resultLineIds = response.jsonPath().getList(".", LineResponse.class).stream()
+                .map(it -> it.getId())
```

Comment:
```suggestion
                .map(LineResponse::getId)
```

가능하다면 `메서드 레퍼런스`를 활용해보는 연습도 좋을 것 같습니다 ~

그리고 저도 예전엔 람다에서 `it` 을 자주 사용했는데, 명확하지 않은 것 같아서 가독성을 위해 좀 더 명확한 네이밍을 하려고 하고있습니다.
참고해주세요~